From 835dc3caff5a97000d0a9b5c5e719ec6f5c4a833 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@nwnk.net>
Date: Mon, 1 Nov 2010 14:29:33 +1000
Subject: [PATCH] Silence useless pedantry warning from find(1) when running cleanlinks.

"find: warning: you have specified the -depth option after a non-option
argument -type, but options are not positional (-depth affects tests
specified before it as well as those specified after it).  Please specify
options before other arguments."

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 cleanlinks |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cleanlinks b/cleanlinks
index 3c81d9f..2724800 100644
--- a/cleanlinks
+++ b/cleanlinks
@@ -22,5 +22,5 @@ find . -type l -print |
 
 echo Removing empty directories ...
 #find . -type d -depth -print | xargs rmdir > /dev/null 2>&1
-find . -type d -depth -empty -print -exec rmdir {} \;
+find . -depth -type d -empty -print -exec rmdir {} \;
 exit 0
-- 
1.7.3.1

